ratingservice/src/main/java/kostka/ratingservice/model/Rating.java

```diff
@@ -0,0 +1,55 @@
+package kostka.ratingservice.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "rating")
+public class Rating {
+    @Id
+    @GeneratedValue
+    private Long ratingId;
+    private Long movieId;
+    private Long userId;
+    private int ratingValue;
+
+    public Rating(final Long ratingId, final Long movieId, final Long userId, final int ratingValue) {
```

Comment:
toto si myslim, ze zase nema smysl. Chybi tam to prirazeni do properties objektu.